src/ontology/mondo.Makefile

```diff
@@ -504,8 +516,17 @@ tmp/mass_obsolete_me.txt: tmp/mass_obsolete.sparql
 mass_obsolete_warning: tmp/mass_obsolete_warning.sparql
 	$(ROBOT) verify -i $(SRC) --queries $< --output-dir reports/
 
-mass_obsolete2: tmp/mass_obsolete.ru tmp/mass_obsolete_me.txt
-	echo "Make sure you have updated config/obsolete_me.txt before running this script.."
+tmp/identify_existing_obsoletes.ru: ../sparql/reports/check_obsoletes_from_list.ru config/obsolete_me.txt
+	@echo "\n** Identify existing obsoletes in config/obsolete_me.txt **"
+	LISTT="$(shell paste -sd" " config/obsolete_me.txt)"; sed "s/MONDO:0000000/$$LISTT/g" $< > $@
+
+.PHONY: identify_existing_obsoletes
+identify_existing_obsoletes: tmp/identify_existing_obsoletes.ru
+	@echo "\n** Write existing obsoletes to tmp/identify_existing_obsoletes.txt **"
+	$(ROBOT) query --format txt -i $(SRC) --queries $< --output-dir tmp/
```

Comment:
There is no need for this goal to be phony:

```suggestion
	$(ROBOT) query --format txt -i $(SRC) --query $< $@
```

Reply:
(we should use .PHONY goals only for workflows that do not result in the creating of a file)

Reply:
For the change to `--query $< $@` - I see why you changed to `--query`, I used queries following the mass_obsoletion_warning format. However, adding in `$< $@` creates a file `identify_existing_obsoletes`. What is the advantage of this vs. writing the file to "tmp" as in the original?